### PR TITLE
Add new grr_fetch_files recipe

### PR DESCRIPTION
**Pull request by tomchop**

### CODE REVIEW

File: dftimewolf/cli/recipes/grr_fetch_files.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    'short_description': _short_description,
+    'modules': [{
+        'name': 'GRRFileCollector',
+        'args': {
+            'hosts': '@hosts',
+            'reason': '@reason',
+            'grr_server_url': '@grr_server_url',
+            'grr_username': '@grr_username',
+            'grr_password': '@grr_password',
+            'files': '@files',
```
Comment (Onager) [inline dftimewolf/cli/recipes/grr_fetch_files.py]: These are full paths to files, right? Do you need TSK:/ prefixes? Something else?

File: dftimewolf/cli/recipes/grr_fetch_files.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+"""Fetches specific files from a GRR host."""
+from __future__ import unicode_literals
+
+_short_description = 'Fetches specific files from a GRR host.'
```
Comment (Onager) [inline dftimewolf/cli/recipes/grr_fetch_files.py]: This works for fetching files from multiple hosts too, please fix

File: dftimewolf/cli/recipes/grr_fetch_files.py (hunk 1 above)
Comment (tomchop) [reply dftimewolf/cli/recipes/grr_fetch_files.py]: No need. I added a clarification about this parameter supporting GRR variable interpolation.

File: dftimewolf/cli/recipes/grr_fetch_files.py (hunk 2 above)
Comment (tomchop) [reply dftimewolf/cli/recipes/grr_fetch_files.py]: Done.

**Pull request merged**